test/runtests.jl

```diff
@@ -209,7 +209,7 @@ test_show(gm17)
 srand(1)
 X = sprand(1000, 10, 0.01)
 β = randn(10)
-y = Bool[rand() < x for x in logistic(X * β)]
+y = Bool[rand() < x for x in @compat logistic.(X * β)]
```

Comment:
More simply and more efficiently written as `rand() < logistic(x)`? (Not that it matters a lot...)